src/content/docs/en/guides/astro-db.mdx

```diff
@@ -8,10 +8,11 @@ import { FileTree } from '@astrojs/starlight/components';
 import PackageManagerTabs from '~/components/tabs/PackageManagerTabs.astro';
 import ReadMore from '~/components/ReadMore.astro';
 import Since from '~/components/Since.astro';
-import StudioHeading from '~/components/StudioHeading.astro';
 import { Steps } from '@astrojs/starlight/components';
 
-Astro DB is a fully-managed SQL database designed exclusively for Astro. Develop locally or connect to a hosted database managed on our [Astro Studio](#astro-studio) platform.
+Astro DB is a fully-managed SQL database designed for the Astro ecosystem: develop locally in Astro and deploy to any libSQL-compatible database.
+
+Astro DB is a complete solution to configuring, developing and querying your data. A local database is created whenever you run `astro dev` to manage your data without the need for Docker or a network connection.
```

Comment:
```suggestion
Astro DB is a complete solution to configuring, developing and querying your data. A local database is created in the `.astro` directory whenever you run `astro dev` to manage your data without the need for Docker or a network connection.
```
I wonder if we should not specify here or elsewhere that in development mode it is also a `.db`. But now that the location is specified, it remains easy for the user to check the format. So it may not be useful.

Reply:
I think here is good! I'll commit it, and if anyone has further suggestion, they can suggest something else! 🙌